handbook/communication/index.md

```diff
@@ -84,6 +84,17 @@ See "[1-1 meetings](../leadership/1-1.md)".
 
 See "[Announcements](announcements.md)".
 
+## Shortening URLs
+Using shorter URLs is much easier to fit into small spaces or images than the full link to our page or handbook. 
```

Comment:
I _think_ you need an extra line spacing between the header and the text, but this might not be 100% necessary.

```suggestion

Using shorter URLs is much easier to fit into small spaces or images than the full link to our page or handbook. 
```